test/system/src/test/java/io/pravega/test/system/ReadWithAutoScaleTest.java

```diff
@@ -87,22 +92,39 @@ public static void setup() throws Exception {
 
         //get the zk ip details and pass it to bk, host, controller
         //2, check if bk is running, otherwise start, get the zk ip
-        Service bkService = new BookkeeperService("bookkeeper", zkUris.get(0));
+        Service bkService = Utils.isDockerLocalExecEnabled() ?
```

Comment:
This could also be made into a factory method

Reply:
Done